src/v/security/audit/audit_log_manager.h

```diff
@@ -46,6 +46,8 @@ concept InheritsFromOCSFBase
 
 class audit_sink;
 
+using is_started = ss::bool_class<struct is_started_t>;
```

Comment:
the `_t` is usually brief for "type", probably what you wanted to write was "is_started_tag"